src/main/java/ru/yandex/practicum/filmorate/exception/FilmorateExceptionHandler.java

```diff
@@ -0,0 +1,28 @@
+package ru.yandex.practicum.filmorate.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class FilmorateExceptionHandler {
+
+    @ExceptionHandler(NotFoundException.class)
```

Comment:
Чтобы корректно обрабатывать все исключения надо также ловить ошибки от валидации и возвращать корректный код ответа:
```suggestion
     @ExceptionHandler({NotFoundException.class, MethodArgumentNotValidException.class})
```